library/src/G4CMPKaplanQP.cc

```diff
@@ -433,6 +440,21 @@ G4CMPKaplanQP::CalcQPAbsorption(G4double qpE,
   return EDep;
 }
 
+
+// Handle quasiparticle energy-dependent absorption efficiency
+
+G4double
+G4CMPKaplanQP::CalcQPEfficiency(G4double qpE) const{
+    G4double eff = absorberEff + absorberEffSlope * qpE;
+
+  if (verboseLevel>2) {
+    G4cout << " efficiency " << eff << G4endl;
```

Comment:
Change "efficiency" to "CalcQPEfficiency", so we know what's being reported when we turn on this debugging output.